src/components/customDataDownload/CustomDataDownload/CustomDataDownload.js

```diff
@@ -532,7 +563,9 @@ const mapDispatchToProps = (dispatch) => {
       dispatch(loadFilterMapping(dataType, dataSubType, years)),
     updateTimePeriodDispatcher: (timePeriod) =>
       dispatch(updateTimePeriod(timePeriod)),
-      hideNavDispatcher: (boolean) => dispatch(hideNav(boolean)),
+    hideNavDispatcher: (boolean) => dispatch(hideNav(boolean)),
+    authApiStatusDispatcher: (status) => dispatch(authApiStatus(status)),
```

Comment:
This function takes zero arguments:
```suggestion
    authApiStatusDispatcher: () => dispatch(authApiStatus()),
```